backend/battle/templates/battle/user/signup_form.html

```diff
@@ -0,0 +1,227 @@
+{% load static %}
+<!DOCTYPE html>
```

Comment:
Any reasoning for re-declaring the whole html again instead of using a base html file and extends?

Note your base.html probably shouldn't be inside `battle/base.html` as well.

Reply:
Nop, it was wrong, thanks :)

Reply:
This is also unsolved